Example/ViewController.swift

```diff
@@ -444,6 +446,15 @@ extension ViewController: MGLMapViewDelegate {
     func mapView(_ mapView: MGLMapView, lineWidthForPolylineAnnotation annotation: MGLPolyline) -> CGFloat {
         return annotation == trackPolyline || annotation == rawTrackPolyline ? 4 : 1
     }
+    
+    func mapView(_ mapView: MGLMapView, regionDidChangeWith reason: MGLCameraChangeReason, animated: Bool) {
+        if foundAllBuildings == false, let navMapView = mapView as? NavigationMapView, reason != .programmatic {
```

Comment:
Would it be problematic for this view controller to react to any camera change, not just programmatic ones? `MGLMapViewDelegate.mapView(_:regionDidChangeWith:animated:)` is an undocumented method that may be removed from future map SDK versions. ViewController is only in the example applications in this repository, but we would need to avoid using this method in any developer-facing examples in navigation-ios-examples.

Reply:
Changed this to `mapViewRegionIsChanging(_ mapView:)`